org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/util/JRTUtil.java

```diff
@@ -534,8 +507,8 @@ class JrtFileSystem {
 	private final Map<String, List<String>> packageToModules = new HashMap<>();
 
 	FileSystem fs;
```

Comment:
Btw. I noticed that this field is overwritten in the `JrtFileSystemWithOlderRelease` sub-class with the CtSystem, but `modRoot` isn't.
I assume that's all intentional? (I'm not very familiar with that code, I just wanted to make sure all is as it should be:) )

Reply:
Yes, it kind of intentional. The `JrtFileSystemWithOlderRelease` doesn't care about it because it overwrites the only method in superclass that uses this field. This isn't optimal, but for now we can leave it.